app/views/favorites/index.html.erb

```diff
@@ -0,0 +1,21 @@
+
+<div class="content">
+
+  <h1 class="header">Favorite Pets</h1>
+
+  <% @favorite_pets.each do |pet| %>
+  <div class="pet-list">
+
+      <img class="pet-image-small" src="<%= pet.image %>" alt="<%= pet.name %> image"> <p>
+      <h2><a id="<%= pet.name %>"class="pet-link hover" href="/pets/<%= pet.id %>"><%= pet.name %></a></h2>
+      <br>
+      <section id="delete-<%= pet.id %>">
+        <%= button_to "Remove #{pet.name} from favorites", "/favorites/#{pet.id}?page=favorites", method: :delete %>
+      </section>
```

Comment:
here is an example of the query params. It works a treat, assuming its acceptable it could be really useful going forward